src/middlewares/__tests__/lambdaIsWarmingUp.js

```diff
@@ -0,0 +1,45 @@
+const middy = require('../../middy')
+const lambdaIsWarmingUp = require('../lambdaIsWarmingUp')
+
+describe('🥃 Lambda Is Warming Up', () => {
+  test(`Should exit with 'warmup' if config.isWarmingUp boolean is provided`, () => {
+    const handler = middy((event, context, cb) => {
```

Comment:
This test needs to be changed considering that the `isWarmingUp` should be a function accepting `event`.

So we should not have an empty `event` object but an `event` with a field (e.g. `warmup: true`) from which we can deduct if the current invocation is for warm up or for actual business logic execution.